fluid/ocr_recognition/scripts/README.md

```diff
@@ -0,0 +1,46 @@
+## Introduction
+Scripts enclosed in the folder serve as examples of commands that start training
+and inference of a model, and are subject to further customisation.
+
+# Running with MKL-DNN
+In order to run training or inference using MKL-DNN library, please use
+`FLAGS_use_mkldnn=1` environmental variable.
+
+## Prerequisites
+In order to run the training and inference, no special requirements are posed.
+
+## Training
+To run training on *CPU*, please execute:
+
+```sh
+source train_cpu.sh
+```
+
+To run training on *CPU* with MKL-DNN, please execute:
+
+```sh
+source train_mkldnn.sh
+```
+
+To run training on *GPU*, please execute:
+
+```sh
+source train_gpu.sh
+```
+
+## Inference
+To perform inference on the trained model using *CPU*, please run:
+```sh
+source infer_cpu.sh
+```
+
+To perform inference on the trained model using *CPU* with MKL-DNN, please run:
+```sh
+source infer_mkldnn.sh
+```
+
+To perform inference on the trained model using *GPU*, please run:
+
+```sh
+source infer_gpu.sh
```

Comment:
Thank you for your suggestion.
The sole purpose of scripts was to give an example of a command allowing for training and inference of a given topology (as described [here](https://github.com/PaddlePaddle/models/pull/1046/files#diff-5183f87f9d91d8b6234d09ccf43ad5e4R2)), while the same time keeping it as simple as possible. Making them more generic would kill the simplicity.

Reply:
Yes, we can understand your point, but we thought it would be very nice to use one script to control all kernels(GPU,CPU, and MKLDNN), batch size and some other tuning parameters. That would make our validation work simple and can prevent some unnecessary tuning error by human factor.

Reply:
The scripts have been adjusted according to your request.